rails-kickoff-template.rb

```diff
@@ -40,13 +40,18 @@ def run_template!
   setup_linters
 
   setup_javascript
+  setup_generators
 
   setup_readme
   create_database
 
   fix_bundler_binstub
 
+  setup_webpacker
+
   output_final_instructions
+
+  exit
```

Comment:
why did you add exit here? there is an additional step if this is being run on an existing application that this is skipping